girder_annotation/girder_large_image_annotation/models/annotation.py

```diff
@@ -679,6 +733,7 @@ def _migrateACL(self, annotation):
         return annotation
 
     def createAnnotation(self, item, creator, annotation, public=None):
+        print('\nCREATING ANNOTATION\n')
```

Comment:
If you want to keep log messages, you should be able to use `logger.debug` (or another appropriate level).  For development, I'll adjust the level to info and then set it to debug once I think it is stable.

Reply:
Removed in 24115d5